<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="MIINDB066XT2" failures="1" tests="8" name="restful-booker-test" time="27.069" errors="0" timestamp="2022-07-11T18:11:04 IST">
  <testcase classname="api.tests.BookingTest" name="createBooking" time="1.775"/>
  <testcase classname="api.tests.BookingTest" name="deleteBooking" time="3.652"/>
  <testcase classname="api.tests.BookingTest" name="getBooking" time="2.645"/>
  <testcase classname="api.tests.BookingTest" name="updateBooking" time="4.092"/>
  <testcase classname="api.tests.MultipleBookingTests" name="getBooking" time="7.981"/>
  <testcase classname="api.tests.NegativeTests" name="createBooking" time="1.441">
    <failure type="java.lang.AssertionError" message="expected [400] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [200]
at api.tests.NegativeTests.createBooking(NegativeTests.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createBooking -->
  <testcase classname="api.tests.NegativeTests" name="getBooking" time="1.459"/>
  <testcase classname="api.tests.PartialUpdateVerification" name="updateBooking" time="3.959"/>
  <testcase classname="api.tests.BaseTest" name="@BeforeSuite setAuthToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setAuthToken -->
  <testcase classname="api.tests.BaseTest" name="@BeforeSuite setAuthToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setAuthToken -->
  <testcase classname="api.tests.BaseTest" name="@BeforeSuite setAuthToken" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setAuthToken -->
</testsuite> <!-- restful-booker-test -->
